tests/unit/plugins/modules/test_user.py

```diff
@@ -127,7 +127,7 @@ def test_change_account_locked_true_bool(self):
         self.module.run_command.return_value = (rc, stdout, stderr)
 
         user.modify_user(self.module)
-        self.module.run_command.assert_called_once_with("chuser account_locked=true  test123")
+        self.module.run_command.assert_called_once_with("chuser account_locked=\"true\"  test123")
```

Comment:
so these parameters `account_locked` (and others) will worked with values that are in quotations?